x/nft/username/codec.proto

```diff
@@ -23,19 +19,21 @@ message ChainAddress {
 message IssueTokenMsg {
     bytes id = 1;
     bytes owner = 2;
-    repeated nft.ActionApprovals approvals = 3 [(gogoproto.nullable) = false];
-    TokenDetails details = 4 [(gogoproto.nullable) = false];
+    repeated bytes approvals = 3;
```

Comment:
In the end, this still requires a field on all the NFTs.

Just changing `nft.ActionApproval`, with all the extra logic
into raw `bytes`, which is a `Condition`, where we try to cram all the extra info in, by overloading fields. And don't actually achieve some generic "approval system" that works like the conditions.